scoring/templates/scoring/council.html

```diff
@@ -74,15 +74,14 @@ <h3 class="mb-3">About this council</h3>
         <div class="col-12 col-lg-5">
             <div class="about-section-child bg-grey-gradient-darker p-3">
                 <h3 class="mb-3">About the scoring</h3>
-                <!-- TODO: New text -->
-                <p>Amet minim mollit non deserunt 
-                    ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
-                </p>
-                <p>Amet minim mollit non deserunt 
-                    ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
-                </p>
+                <p>Climate Emergency UK assessed plans according to 28 questions across nine sections, based on the expert-approved <a class="d-inline-block" href="https://www.climateemergency.uk/blog/climate-action-plan-checklist/">checklist for Council Action Plans</a>. Each council was marked against these criteria and given a right to reply before the scores underwent a final audit. This work was completed between June and December 2021. Action Plans and related documents that were publicly available after September 2015 and before 20 September 2021 were marked. </p>
+                {% comment %} TODO the following paragraphs need a conditional to evaluate the following {% endcomment %}
```

Comment:
We’ll be able to do this once https://github.com/mysociety/caps/pull/264 has been merged. /cc @struan